app/Http/Controllers/LegacyController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Legacy;
+use Illuminate\Support\Facades\Lang;
+
+class LegacyController extends Controller
```

Comment:
@abm-devmonk-purple Is this Controller temporary?

Reply:
Yes.